app/components/featured_partnerships_component/featured_partnerships_component.scss

```
@@ -0,0 +1,42 @@
+@import "variables_mixins";
+
+.featured_partnerships {
+	margin-bottom: 2rem;
+}
+
+.featured_partnerships--title {
+	color: $home-background-3;
+	font-family: $sans-serif;
+	font-size: 2.22222rem;
+	margin: 0;
+	text-align: center;
+}
+
+.featured_partnerships--container {
+	display: flex;
+	gap: 1rem;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+
+	@include for-tablet-portrait-up {
+		flex-direction: row;
+		flex-wrap: wrap;
+	}
+
+	@include for-tablet-landscape-up {
+		flex-wrap: nowrap;
+	}
+}
+
+.featured_partnerships--item {
+	width: 20rem;
```

```suggestion
       max-width: 20rem;
```

We're getting overflow at very small screen sizes but it's a quick fix